### PR TITLE
mgmt, elasticsan, add live tests

### PR DESCRIPTION
**Pull request by v-hongli1**
relate to [issue#35316](https://github.com/Azure/azure-sdk-for-java/issues/35316)

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/elasticsan/azure-resourcemanager-elasticsan/src/test/java/com/azure/resourcemanager/elasticsan/ElasticSanManagerTests.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    public void testCreateElasticSan() {
+        ElasticSan elasticSan = null;
+        try {
+            String elasticSanName = "elasticsan" + randomPadding();
+            // @embedStart
+            elasticSan = elasticSanManager.elasticSans()
+                .define(elasticSanName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withSku(new Sku().withName(SkuName.PREMIUM_LRS).withTier(SkuTier.PREMIUM))
```
Comment (weidongxu-microsoft) [inline sdk/elasticsan/azure-resourcemanager-elasticsan/src/test/java/com/azure/resourcemanager/elasticsan/ElasticSanManagerTests.java:82]: Do we have lower tier? Free or Basic? (there is a cost on these resource)

Also, use lower TB below (1 or 2?)

Comment (v-hongli1) [reply sdk/elasticsan/azure-resourcemanager-elasticsan/src/test/java/com/azure/resourcemanager/elasticsan/ElasticSanManagerTests.java:82]: 1. The `ElasticSan` only supports `Premium_LR` and `Premium_ZRS`, `Free` and `Basic` cannot be used.
2. `Also, use lower TB below (1 or 2? 24 is larger than all my SSD + HD combined)` has been fixed in the new version.

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**